File: java/src/main/java/com/sdl/web/pca/client/Schema/GraphQLSchemaTypeInfo.java
package com.sdl.web.pca.client.schema;

public class GraphQLSchemaTypeInfo {
    public String Name;
    public String Kind;
    public GraphQLSchemaTypeInfo OfType;

    public void setName(String name) {
        Name = name;
    }

    public String getName() {
        return Name;
    }

    public void setKind(String kind) {
        Kind = kind;
    }

    public String getKind() {
        return Kind;
    }

    public void setOfType(GraphQLSchemaTypeInfo ofType) {
        OfType = ofType;
    }

    public GraphQLSchemaTypeInfo getOfType() {
        return OfType;
    }
}

File: net/src/Sdl.Tridion.Api.Client/GraphQLClient/Schema/GraphQLSchemaTypeInfo.cs
namespace Sdl.Tridion.Api.GraphQL.Client.Schema
{
    public class GraphQLSchemaTypeInfo
    {
        public string Name { get; set; }
        public string Kind { get; set; }
        public GraphQLSchemaTypeInfo OfType { get; set; }
    }
}
